From ee38888b1c62daba63b88caa891fba80915c6381 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 3 Jan 2017 15:45:37 -0800
Subject: [PATCH] IMGAPI-609 minor improvements to adminimportimage.dc-test.js
 test case messages Reviewed by: Orlando Vazquez <orlando@joyent.com> Approved
 by: Orlando Vazquez <orlando@joyent.com>

---
 test/adminimportimage.dc-test.js | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/test/adminimportimage.dc-test.js b/test/adminimportimage.dc-test.js
index 8935977..9887edd 100644
--- a/test/adminimportimage.dc-test.js
+++ b/test/adminimportimage.dc-test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2017 Joyent, Inc.
  */
 
 /*
@@ -146,16 +146,18 @@ test('AdminImportImage from local .imgmanifest', function (t) {
 
     function create(next) {
         self.client.adminImportImage(data, function (err, image, res) {
-            t.ifError(err, err);
-            t.ok(image);
+            t.ifError(err);
+            t.ok(image, 'got an image object');
             if (image) {
-                t.equal(image.uuid, data.uuid);
-                t.equal(image.published_at, data.published_at);
-                t.equal(image.state, 'unactivated');
+                t.equal(image.uuid, data.uuid, 'expected image.uuid matched');
+                t.equal(image.published_at, data.published_at,
+                    'expected image.published_at matched');
+                t.equal(image.state, 'unactivated',
+                    'image.state is unactivated');
                 t.equal(image.inherited_directories.join(','),
                     data.inherited_directories.join(','),
-                    'inherited_directories');
-                t.ok(image.billing_tags);
+                    'expected inherited_directories matched');
+                t.ok(image.billing_tags, 'have image.billing_tags');
             }
             next(err);
         });
-- 
2.21.0

